lib/pipeline_controller.py

```diff
@@ -73,12 +56,7 @@ def __init__(self, now):
 
         self.census_geocoder_client = CensusGeocoderApiClient()
         self.nyc_geocoder_client = NycGeocoderClient()
-        self.s3_client = S3Client(
-            os.environ['S3_BUCKET'], os.environ['S3_RESOURCE'])
         self.avro_encoder = AvroEncoder(os.environ['PATRON_INFO_SCHEMA_URL'])
-        self.kinesis_client = KinesisClient(
-            os.environ['KINESIS_STREAM_ARN'],
-            int(os.environ['KINESIS_BATCH_SIZE']))
```

Comment:
In relation to my previous comment -- why have the KinesisClient & S3Client been removed? And since they are removed, shouldn't their corresponding configs in production.yaml and qa.yaml be empty too?